HHS/Head-Start-TTADP
# [PROD] 07/10 TTAHUB-3102 Temporarily disable resource scrape job, TTAHUB-2907 Add a "something went wrong" page

## Description of change

**TTAHUB-3102:** We will temporarily disable the resource scrape job to see if it has any effect on the Sequelize connection timeout.
**TTAHUB-2907**: Added a generic page to handle the various errors we might get in the HUB.

## How to test

**TTAHUB-3102:** Nothing to test creating reports should work as before.
**TTAHUB-2907**: Visit a page that does not exist in the HUB should see the new page.

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-3102
* https://jira.acf.gov/browse/TTAHUB-2907

## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
